imports/mail/RoleChangeMailHandler.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Created by Simon on 25.05.2017.
+ */
+import { Meteor } from 'meteor/meteor';
+
+import { MailFactory } from './MailFactory'
+import { GlobalSettings } from '../config/GlobalSettings'
+import {UserRoles as userroles} from "../userroles";
+import {MeetingSeries} from "../meetingseries";
+
+export class RoleChangeMailHandler {
+    constructor(userId, oldRole, newRole, moderator, meetingSeriesId) {
+        this._oldRole = oldRole;
+        this._newRole = newRole;
+        this._moderator = moderator;
+        this._meetingSeriesId = meetingSeriesId;
+        this._user = Meteor.users.findOne(userId);
+        if (!this._user) {
+            throw new Meteor.Error("Send Role Change Mail", "Could not find user: "+ userId);
+        }
+    }
+
+    send() {
+        let emailFrom = this._moderator.emails;
+        let adminFrom = (emailFrom && emailFrom.length > 0)
+            ? emailFrom[0].address
+            : GlobalSettings.getDefaultEmailSenderAddress();
+        let emailTo = this._user.emails[0].address;
+
```

Comment:
Please check, why if a moderator has a correct eMail in userdfatabase his mail address is not the "FROM" address of the mail. Maybe this is a bug in our existing code already? But this whole block does not evailuate to the moderator mail address.... Please check.